src/viewport/viewport.model.ts

```diff
@@ -1,23 +1,14 @@
-export interface ViewportSize {
-	width: number;
-	height: number;
-}
+import { Dictionary } from "../internal/internal.model";
 
-export enum ViewportSizeType {
-	xsmall = 0,
-	small = 1,
-	medium = 2,
-	large = 3,
-	xlarge = 4,
-	xxlarge = 5,
-	xxlarge1 = 6
-}
-
-export interface ViewportSizeTypeInfo {
-	type: ViewportSizeType;
-	name: string;
-	widthThreshold: number;
-}
+export const UxViewportSizes = {
+	xsmall: 450,
+	small: 767,
+	medium: 992,
+	large: 1200,
+	xlarge: 1500,
+	xxlarge: 1920,
+	xxlarge1: 2100
+};
```

Comment:
default viewport sizes provided by the library

Reply:
can you rename it as it was so its less of a breaking change

Reply:
I think it's best if it remains as is because every `ViewportSizeType` would still need to be revised since every property represents the width rather than the position.

consider this scenario which is a tap from the observable `sizeType$`;
`ViewportSizeTypeInfo.type < ViewportSizeType.medium`

it will no longer be valid as `.type` represents the position/order while the `.medium` no longer represents the order but the width threshold. 
It will still work if adjusted to `ViewportSizeTypeInfo.widthThreshold < ViewportSizeType.medium`.
___
anyway, I've reverted it back to `ViewportSizeType` but thought I may raise this

Reply:
Reverted back to `UxViewportSizes`, and to replace the enum `ViewportSizeType`, I've created a util method `generateViewportSizeType` which accepts the custom breakpoints and returns a name/position object.
This would need to be generated and exposed globally on the application side by the user;
```
import { generateViewportSizeType } from "@ssv/ngx.ux";
const breakpoints = {
    small: 300,
    medium: 500,
    large: 800,
    fullhd: 1920,
}
export const ViewportSizeType = generateViewportSizeType(breakpoints);
```